packages/keystone/test.utils.js

```diff
@@ -373,11 +373,21 @@ const makeLoggedInAdminClient = async () => {
     return await makeLoggedInClient({ email: DEFAULT_TEST_ADMIN_IDENTITY, password: DEFAULT_TEST_ADMIN_SECRET })
 }
 
+/**
+ * Used for async action waiting in tests. Pass in callback and options.
+ * Will retry executing callback again and again each $interval ms for $timeout ms.
```

Comment:
You don't need to change this core function for your case. We already have a timeout argument. If you want to execute some function less than some count of time then you need to pass the right timeout argument.

Reply:
reverted